Added a note about including a deploy_token (#871)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

<Lightbox src="/img/docs/dbt-cloud/connecting-gitlab/connected.png" title="GitLab connected" />

> Please note, you will need to ensure your deploy token is populated in your project settings page, in addition to your profile page. 

Reviewer: ```suggestion
Now that you've linked to your account from your Profile page, you should verify that you see your deploy token in your Project Settings page. 
```

> Now that you've linked to your account from your Profile page, you should verify that you see your deploy token in your Project Settings page. 

Reviewer: ```suggestion
Now that you've linked to your account from your Profile page, you should verify that your Project Settings page also shows your deploy token. 
```
